lxc/exec.go

```diff
@@ -20,10 +22,29 @@ func (c *execCmd) usage() string {
 	return gettext.Gettext(
 		"Execute the specified command in a container.\n" +
 			"\n" +
-			"lxc exec container [command]\n")
+			"lxc exec container [--env EDITOR=/usr/bin/vim]* [command]\n")
```

Comment:
Is that the standard syntax? I've always used:

    lxc exec container [--env EDITOR=abc]... [command]

To denote that an option may be used multiple times.

Also, I believe the command is required nowadays, so we should make that \<command\>